api/router/router.go

```diff
@@ -0,0 +1,19 @@
+package router
+
+import (
+	c "api/controller"
+
+	"github.com/gorilla/mux"
+	"github.com/unrolled/render"
+)
+
+var Resp = render.New()
+
+func NewRouter() *mux.Router {
+	router := mux.NewRouter().StrictSlash(true)
+
+	router.HandleFunc("/api", c.HelloWorld)
```

Comment:
If you want to add a prefix to all of your routes, you could create a router that already has that prefix, that way you could avoid duplicating the same thing

`router = router.PathPrefix("/api").Subrouter()`